lib/bolt/cli.rb

```diff
@@ -190,7 +190,13 @@ def create_option_parser(results)
         end
         opts.on('--sudo-password [PASSWORD]',
                 'Password for privilege escalation') do |password|
-          options[:sudo_password] = password
+          if password.nil?
+            STDOUT.print "Please enter your privelege escalation password: "
```

Comment:
privilege

Reply:
I managed to misspell that everywhere, fixed up.